functions/source/soci-index-generator-lambda/handler.go

```diff
@@ -291,38 +324,47 @@ func buildIndex(ctx context.Context, dataDir string, sociStore *store.SociStore,
 	if err != nil {
 		return nil, err
 	}
-
-	builder, err := soci.NewIndexBuilder(containerdStore, sociStore, artifactsDb, soci.WithPlatform(platform))
-	if err != nil {
-		return nil, err
+	builderOpts := []soci.BuilderOption{
+		//soci.WithMinLayerSize(0),
+		soci.WithBuildToolIdentifier("AWS SOCI CLI v0.2"),
```

Comment:
Can we change this to `AWS SOCI Index Builder Cfn v0.2` or something instead? I don't think we should have used the SOCI CLI build tool identifier originally.